src/shared/components/oncoprint/controls/styles.scss

```diff
@@ -37,6 +36,10 @@
     }
   }
 
+  .dropdown-menu.heatmap {
```

Comment:
use sass & syntax to nest this in the .dropdown-menu class above.

Reply:
Implemented your change.